test/schemas/testpilot.js

```diff
@@ -1,41 +1,51 @@
 /* global expect */
 "use strict";
 
-const Joi = require("joi");
+const Joi = require("joi-browser");
 const PingCentre = require("../../ping-centre");
 const testPilotSchema = require("../../schemas/testpilot.js");
 
 const samplePing = {
-  type: "testpilot",
-  id: "20076a92-360c-da4d-9742-8327848c5070",
-  creationDate: "2016-11-30T22:46:57.230Z",
-  version: 4,
-  payload: {
-    timestamp: 483,
-    test: "@testpilot-addon",
-    version: "0.9.1-dev-e42d9cb",
-    events: [{
-      timestamp: 483,
-      event: "clicked",
-      object: "txp_toolbar_menu_1"
-    }]
-  },
-  clientId: "729e4a78-df2e-c347-ba29-18f34e5d5759",
-  // To keep the sample ping reasonably small, I've replaced the deeply nested
-  // application and environment properties with empty objects.
-  application: {},
-  environment: {},
-  // These fields are in the common schema, and are normally added to the
-  // ping by ping-centre. Adding them manually so the test ping will validate.
   topic: "testpilot",
   client_id: "729e4a78-df2e-c347-ba29-18f34e5d5759",
-  event_type: "test"
+  event_type: "clicked",
+  object: "txp_toolbar_menu_1",
+  client_time: 483,
+  firefox_version: "50.0.1",
+  addon_version: "0.9.4",
+  os_name: "Darwin",
+  os_version: "16.0.0",
+  locale: "en-US",
+  raw: JSON.stringify({
+    type: "testpilot",
+    id: "20076a92-360c-da4d-9742-8327848c5070",
+    creationDate: "2016-11-30T22:46:57.230Z",
+    version: 4,
+    payload: {
+      timestamp: 483,
+      test: "@testpilot-addon",
```

Comment:
Oh yeah! Good point.

I was thinking the addon ID was implied, since only the testpilot addon is sending this event--but it actually seems like we could use this for the experiments as well, in which case, having the addon ID is necessary.

I'll add an `addon_id` field, as well as a field to track A/B test info (`variants`), and then I'll go ahead and remove the redundant `testpilottest` schema.